### ISSUE TITLE
Summary shows field label negated by condition

### ISSUE THREAD

**Issue by watarutmnh** [verbb/formie#733]
**Description**
Summary field shows field labels those are negated by conditional statement.
They better be hidden since they have no value to submit.

### Form settings
Please pick from the options below.

- Multi-page form: Yes
- Submission Method: Page Reload
- Client-side Validation: No
- Custom Form Templates: No

**Additional info**
- Plugin version: dev-craft-3 as 1.5.7
- Craft version: 3.7.30.1
- Multi-site: Yes

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.7",
  "...": "..."
}
```

Then run `composer update`.

**Comment by watarutmnh** [comment]
@engram-design I confirmed the fix.
Thank you!

**Comment by engram-design** [comment]
Fixed in [1.5.8](https://github.com/verbb/formie/releases/tag/1.5.8)
